assets/app/views/settings.html

```diff
@@ -29,91 +29,153 @@
-          </th>
-          <th>
-            Max
-            <i class="small pficon pficon-help" data-toggle="tooltip" data-placement="right"
-          title="The maximum amount of this compute resource that can be requested.  The limit must also be below the maximum value."></i>
-          </th>
-          <th>
-            Default Request
-            <i class="small pficon pficon-help" data-toggle="tooltip" data-placement="right"
-          title="A container will default to request this amount of a compute resource if no request is specified. The system will guarantee the requested amount of compute resource when scheduling a container for execution. If a quota is enabled for this compute resource, the quota usage is incremented by the requested value."></i>
-          </th>
-          <th>
-            Default Limit
-            <i class="small pficon pficon-help" data-toggle="tooltip" data-placement="right"
-          title="The default limit defines the maximum amount of compute resource the container may have access to during execution if no limit is specified. If no request is made for the compute resource on the container or via a Default Request value, the container will default to request the limit."></i>
-          </th>
-          <th>
-            Max Limit/Request Ratio
-            <i class="small pficon pficon-help" data-toggle="tooltip" data-placement="right"
-          title="If specified, the compute resource must have a request and limit that are both non-zero, where limit divided by request is less than or equal to the specified amount; this represents the max burst for the compute resource during execution."></i>
-          </th>
-        </thead>
-        <tbody ng-repeat="(limitRangeName, limitRange) in limitRanges">
-          <tr ng-repeat-start="limit in limitRange.spec.limits"></tr>
-          <tr ng-repeat="(type, typeLimits) in limitsByType[limitRangeName][limit.type]">
-            <!-- TODO prettify type -->
-            <td>{{limit.type}} {{type}}</td>
-            <td>{{(typeLimits.min | usageWithUnits : type) || "&mdash;"}}</td>
-            <td>{{(typeLimits.max | usageWithUnits : type) || "&mdash;"}}</td>
-            <td>{{(typeLimits.defaultRequest | usageWithUnits : type) || "&mdash;"}}</td>
-            <td>{{(typeLimits["default"] | usageWithUnits : type) || "&mdash;"}}</td>
-            <td>{{typeLimits.maxLimitRequestRatio || "&mdash;"}}</td>
-          </tr>
-          <tr ng-repeat-end></tr>
-        </tbody>
-      </table>
+      <div ng-if="(quotas | hashSize) == 0">
+        <h2>
+          <span>Quota</span>
+          <i style="margin-left: 5px; cursor: help;" class="small pficon pficon-help" data-toggle="tooltip" data-placement="right"
+             dynamic-content="{{quotaHelp}}"></i>
+        </h2>
+        <p><em>{{emptyMessageQuotas}}</em></p>
+      </div>
+      <div ng-repeat="quota in quotas" class="gutter-bottom">
+        <h2>
+          Quota {{quota.metadata.name}}
```

Comment:
debating whether we should only show quota name when there is more than one, a lot of our examples use the name "quota" ...  "Quota quota" is going to show up a lot, like when we had Project Example Project

cc @derekwaynecarr